qa/statsd/GNUmakefile

```diff
@@ -2,14 +2,50 @@ TOPDIR = ../..
 include $(TOPDIR)/src/include/builddefs
 
 TESTDIR = $(PCP_VAR_DIR)/testsuite/statsd
-RBFILES = $(shell echo *.rb)
+TESTSRCDIR = src
+TESTFILES = $(shell find $(TESTSRCDIR) -type f)
 
 ifeq "$(PMDA_STATSD)" "true"
 default setup default_pcp:
 
-install install_pcp: default $(RBFILES)
+install install_pcp: default $(TESTFILES)
 	$(INSTALL) -m 755 -d $(TESTDIR)
-	$(INSTALL) -m 644 $(RBFILES) $(TESTDIR)
+	# I wish I knew how to extract such listing from $(TESTFILES), not sure why this is needed yet
+	$(INSTALL) -m 644 src/cases/01.py $(TESTDIR)/src/cases/01.py
+	$(INSTALL) -m 644 src/cases/02.py $(TESTDIR)/src/cases/02.py
+	$(INSTALL) -m 644 src/cases/03.py $(TESTDIR)/src/cases/03.py
+	$(INSTALL) -m 644 src/cases/04.py $(TESTDIR)/src/cases/04.py
+	$(INSTALL) -m 644 src/cases/05.py $(TESTDIR)/src/cases/05.py
+	$(INSTALL) -m 644 src/cases/06.py $(TESTDIR)/src/cases/06.py
+	$(INSTALL) -m 644 src/cases/07.py $(TESTDIR)/src/cases/07.py
+	$(INSTALL) -m 644 src/cases/08.py $(TESTDIR)/src/cases/08.py
+	$(INSTALL) -m 644 src/cases/09.py $(TESTDIR)/src/cases/09.py
+	$(INSTALL) -m 644 src/cases/10.py $(TESTDIR)/src/cases/10.py
+	$(INSTALL) -m 644 src/cases/11.py $(TESTDIR)/src/cases/11.py
+	$(INSTALL) -m 644 src/cases/12.py $(TESTDIR)/src/cases/12.py
+	$(INSTALL) -m 644 src/cases/13.py $(TESTDIR)/src/cases/13.py
+	$(INSTALL) -m 644 src/cases/14.py $(TESTDIR)/src/cases/14.py
+	$(INSTALL) -m 644 src/cases/15.py $(TESTDIR)/src/cases/15.py
+	$(INSTALL) -m 644 src/configs/complex/0/pmdastatsd.ini $(TESTDIR)/src/configs/complex/0/pmdastatsd.ini
```

Comment:
Likewise for each of these new ini file dirs :|

Since these configs are so small, I'd consider generating them at runtime instead of committing them as individual files in the repo.